rsconnect/main.py

```diff
@@ -244,6 +244,58 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+# inverts bool args if they are provided, otherwise returns None
+def env_management_callback(ctx, param, value):
+    # eval the shorthand flag if it was provided
+    disable_env_management = ctx.params.get('disable_env_management')
+    if disable_env_management is not None:
+        value = disable_env_management
+
+    # invert env_management value if it was provided
+    if value is not None:
+        return not value
+    return value
```

Comment:
Is this equivalent?

```suggestion
    value = ctx.params.get('disable_env_management')
    return (not value) if (value != None) else value
```

Reply:
Not quite; `value` is either `--disable-env-management-r` or `--disable-env-management-py`, so we can't re-assign until we know if `ctx.params.get('disable_env_management') is None`. We also need to check whether `value is None` so that we can handle the null case which means that nothing should be written to the manifest.

```
$ rsconnect write-manifest api --disable-env-management-py ./
env_management_py:  None
env_management_r:  None
$ rsconnect write-manifest api --disable-env-management ./
env_management_py:  False
env_management_r:  False
```

Reply:
Ah, I overlooked `value` being a method argument. Makes sense. Thanks!